docs/conf.py

```diff
@@ -100,7 +100,7 @@ def setup(app):
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
```

Comment:
this raised a warning when generating the docs and defaulted to en, setting it to en prevents the warning.